src/main/java/org/opentripplanner/transit/raptor/rangeraptor/standard/besttimes/BestTimes.java

```diff
@@ -66,21 +67,21 @@ public int transitArrivalTime(int stop) {
    * @return true if at least one stop arrival was reached last round (best overall).
    */
   public boolean isCurrentRoundUpdated() {
-    return !reachedCurrentRound.isEmpty();
+    return reachedCurrentRound.cardinality() > 0;
```

Comment:
This is slow, but should not have a huge impact. We can look for the first bit set or keep a flag in the BestTimes to improve this.